backend/tests/app/app.go

```diff
@@ -0,0 +1,63 @@
+package app
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/sqlite"
+
+	"github.com/lanets/floorplanets/backend/app"
+	"github.com/lanets/floorplanets/backend/models"
+)
+
+type TestApp struct {
+	t       *testing.T
+	tempDir string
+	App     *app.App
+}
+
+func NewTestApp(t *testing.T) *TestApp {
+	tempDir, err := ioutil.TempDir("", "floorplanets-tests")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Setup the database
+	databaseFile := path.Join(tempDir, "testdb.sqlite")
+	database, err := setupDatabase(databaseFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Setup the app
+	application := app.NewApp(database)
+
+	testApp := TestApp{
+		t:       t,
+		tempDir: tempDir,
+		App:     application,
+	}
+
+	return &testApp
+}
+
+func setupDatabase(databaseFile string) (*gorm.DB, error) {
+	database, err := gorm.Open("sqlite3", databaseFile)
```

Comment:
Yes for last comment, but not this one. This one is a test, its created and deleted on the go.